src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -60,6 +60,14 @@ Once you've updated the yaml file, return to this doc to install the dashboards
     </Collapser>
 </CollapserGroup>
 
+## Get Snowflake metrics in a dashboard [#dash]
+
+Once you've installed the [Snowflake quickstart](https://newrelic.com/instant-observability/snowflake), you can see your critical Snowflake data in New Relic. 
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then click **Metrics & events**.
+2. Run either NRQL queries to view your Snowflake metrics: `SELECT * FROM SnowflakeAccount` or `SELECT * FROM SnowflakeVirtualWarehouse`.
+3. Once you've confirmed that New Relic ingested your Snowflake metrics, go to **Dashboards** from **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** to see your Snowflake dashboards. 
```

Comment:
Once you've confirmed that New Relic ingested your Snowflake metrics:
1. go to Add Data from one.newrelic.com -> Search for Snowflake in Dashboards section.
2. Install the Snowflake dashboard.
3. Then you can see your dashboard in [one.newrelic.com](https://one.newrelic.com/all-capabilities) -> ***Dashboards*** section